build_scripts/docker_image_creator.py

```diff
@@ -0,0 +1,68 @@
+""" Script to create Sagemaker TensorFlow Docker image
+    Run the command:
+        python docker_image_creator.py optimized_binary_link gpu|cpu tensorflow_version python_version
+
+"""
+
+import argparse
+import glob
+import subprocess
+import sys
+
+## GLOBALS
+DOCKER = 'docker' # Stores which type of DOCKER to use, nvidia-docker or docker
+PATH_TO_SCRIPT = ''  # Stores route the user takes to build scripts directory
+
+def create_docker_image(optbin_link, processor, framework_version, python_version):
+    """ Function builds a docker image with the TF optimized binary
```

Comment:
No need for a space after `"""`

Reply:
Got it. Just to clarify this is what it should look like correct?
``` python
def create_docker_image(optbin_link, processor, framework_version, python_version):
    """ Function builds a docker image with the TF optimized binary
    Args:
        optbin_link (str): link to where the optimized binary is
        processor (str): gpu or cpu
        framework_version (str): tensorflow version i.e 1.6.0
        python_version (str): (i.e. 3.6.5 or 2.7.4)
    """
    # 1.) Initialize commonly used variables
```

As well as this at the top?
``` python
""" Script to create Sagemaker TensorFlow Docker image
    Run the command:
        python docker_image_creator.py optimized_binary_link gpu|cpu tensorflow_version python_version
"""
import argparse
```

Reply:
you don't need the space after the first `"""`, and add newlines between paragraphs/sections. so they should look like:

```
"""Function to build a Docker image...

Args:
    optbin_link (str): link
    ...
"""
```

and 

```
"""Script to create SageMaker TF images...

Usage:
    python docker_image_creator.py ...
"""
```